src/FSharpVSPowerTools.Logic/Navigation/GoToDefinitionFilter.fs

```diff
@@ -33,6 +28,12 @@ type internal UrlChangeEventArgs(url: string) =
     inherit EventArgs()
     member __.Url = url
 
+[<NoComparisonAttribute; NoEquality>]
+type GoToDefinitionResolution =
```

Comment:
For big project, it makes sense to use qualified names e.g. `GoToDefinitionResult. LoadDirective`. You can add `RequireQualifiedAccess` to DU definition to reinforce that.